src/groups/mqb/mqbblp/mqbblp_queueconsumptionmonitor.t.cpp

```diff
@@ -338,58 +219,19 @@ TEST_F(Test, emptyQueue)
 
     d_monitor.setMaxIdleTime(k_MAX_IDLE_TIME);
 
-    d_monitor.registerSubStream(
-        mqbu::StorageKey::k_NULL_KEY,
-        bdlf::BindUtil::bind(&Test::head, this, mqbu::StorageKey::k_NULL_KEY));
+    d_monitor.registerSubStream(d_storageKey);
 
     d_monitor.onTimer(k_MAX_IDLE_TIME);
-    ASSERT_EQ(d_monitor.state(mqbu::StorageKey::k_NULL_KEY),
+    ASSERT_EQ(d_monitor.state(d_storageKey),
               QueueConsumptionMonitor::State::e_ALIVE);
     ASSERT_EQ(logObserver.records().size(), expectedLogRecords);
 
     d_monitor.onTimer(k_MAX_IDLE_TIME + 1);
-    ASSERT_EQ(d_monitor.state(mqbu::StorageKey::k_NULL_KEY),
+    ASSERT_EQ(d_monitor.state(d_storageKey),
               QueueConsumptionMonitor::State::e_ALIVE);
     ASSERT_EQ(logObserver.records().size(), expectedLogRecords);
 }
 
-TEST_F(Test, logFormat)
```

Comment:
This test originally tested the log format, but now logging is done by the callback, so this test makes no sense.